### DISCUSSION TITLE
Default settings

### DISCUSSION THREAD

**Discussion by IBBoard** [CodedOre/NewCaw#61] [General]
I'm running with GSettings in memory at the moment, so all of the settings keep resetting. I think that some different settings would be more user-friendly.

```diff
diff --git a/data/uk.co.ibboard.Cawbird.gschema.xml b/data/uk.co.ibboard.Cawbird.gschema.xml
index 021955f..ec356f9 100644
--- a/data/uk.co.ibboard.Cawbird.gschema.xml
+++ b/data/uk.co.ibboard.Cawbird.gschema.xml
@@ -6,13 +6,13 @@
       <description>
         Display user avatars round.
       </description>
-      <default>true</default>
+      <default>false</default>
     </key>
     <key type="b" name="trailing-tags">
       <description>
         If to show tags after the text.
       </description>
-      <default>false</default>
+      <default>true</default>
     </key>
     <key type="b" name="internal-links">
       <description>
@@ -24,7 +24,7 @@
       <description>
         If a double click is required to activate posts.
       </description>
-      <default>false</default>
+      <default>true</default>
     </key>
        </schema>
   <!-- Window Sizes -->
```

* Square avatars by default, because avatars SHOULD be square. People make square images, and round images enforce artistic constraints that don't work for everyone (whereas people who want a circular shape can choose to put a circle within a square) - see also: IBBoard/cawbird#56
* Keep the trailing tags - people put the tags in for a reason, so don't hide content unless the user chooses to
* BUT keep the hiding of internal media links, because we're showing those as images
* Require double-click to avoid accidentally navigating when you click between overlapping windows and click on a post. Ideally GTK would have a "don't activate widgets on focus change" option, but I guess there's times when it _is_ what you want

What do other people think?

**Comment by CodedOre** [comment]
> * Square avatars by default, because avatars SHOULD be square. People make square images, and round images enforce artistic constraints that don't work for everyone (whereas people who want a circular shape can choose to put a circle within a square) - see also: IBBoard/cawbird#56

In most places you now find round avatars, on Twitter, on Github, and importantly in Gnome. And since NewCaw is designed following Gnome's HIG, I think the default should follow Gnomes one. 

> * Keep the trailing tags - people put the tags in for a reason, so don't hide content unless the user chooses to

That should definitely be true. I guess this is an oversight from when I changed it from "Hide Trailing Tags".

> * Require double-click to avoid accidentally navigating when you click between overlapping windows and click on a post. Ideally GTK would have a "don't activate widgets on focus change" option, but I guess there's times when it _is_ what you want

I don't have a good opinion on that. Double click does make sense though given basically all content in the window is activatable.

**Reply by IBBoard** [reply]
> In most places you now find round avatars, on Twitter, on Github, and importantly in Gnome. And since NewCaw is designed following Gnome's HIG, I think the default should follow Gnomes one.

That just means that Gnome is wrong, along with everywhere else 😉 I don't understand how graphics designers everywhere have joined this massed delusion that round avatars are better.
